tests/unit/s2n_certificate_extensions_test.c

```diff
@@ -29,120 +29,53 @@
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_safety.h"
 
-#define SIZEOF_UINT24 3
+s2n_cert_public_key public_key;
+s2n_pkey_type actual_cert_pkey_type;
 
-static int s2n_certificate_extensions_send_empty(struct s2n_stuffer *stuffer)
-{
-    struct s2n_connection conn = { 0 };
-    return s2n_extension_list_send(S2N_EXTENSION_LIST_EMPTY, &conn, stuffer);
-}
+static const s2n_parsed_extension empty_extensions[S2N_PARSED_EXTENSIONS_COUNT] = { 0 };
 
-static int s2n_certificate_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *stuffer,
-        struct s2n_cert_chain_and_key *chain_and_key)
+#define EXPECT_EXTENSIONS_EMPTY(extensions) EXPECT_BYTEARRAY_EQUAL(extensions.parsed_extensions, empty_extensions, sizeof(empty_extensions))
+#define EXPECT_EXTENSIONS_NOT_EMPTY(extensions) EXPECT_BYTEARRAY_NOT_EQUAL(extensions.parsed_extensions, empty_extensions, sizeof(empty_extensions))
```

Comment:
These macros could be moved to `s2n_test.h`.

Reply:
Good call! Those are used in multiple tests at this point.